### DISCUSSION TITLE
Enable Maximize Editor Groupe View

### DISCUSSION THREAD

**Discussion by fretwurst** [posit-dev/positron#4154] [Feedback] [answered]
On small laptop screens, I find the VS Code Maximize Editor View function very practical. Is it possible to enable this function "View: Toggle Maximize Editor Group" (workbench.action.minimizeOtherEditors) in Positron for the source file and the viewer?

**Comment by juliasilge** [comment]
Can you say more about what you are not able to do? It looks to me like `workbench.action.minimizeOtherEditors` is working as expected when you have editor groups:

![Screenshot 2024-07-25 at 11 51 29 AM](https://github.com/user-attachments/assets/3afa97f0-b20b-408c-8943-366218185c54)

After I use the command, the other editors are in fact minimized.

**Reply by fretwurst** [reply]
Thank you for trying, and please excuse me, I found a function description that is incorrect. Let me describe it. In VSC and Positron, I can open a qmd file, split the editor and then open the rendered PDF there. When I double-click on "Formelsammlung.pdf" at the top, the file is displayed wider and the other editor group is only narrow on the right side. I can switch between the editor groups with cmd + 1 and cmd + 2, where the focused editor group is automatically displayed wider. 
![Positron activate auto focus](https://github.com/user-attachments/assets/058a5b72-5233-489a-b509-3e2cb26ab675)
![Positron auto focus](https://github.com/user-attachments/assets/3a86409a-f57e-4cbf-80af-fa97593d9444)

In VS Code, after rendering the preview, the rendered file is shown in such an editor. In Positron, however, after rendering, the VIEWER is automatically opened, which is not an editor group. This means that it doesn't automatically adjust the width of the focused windows. But it would be quite cool if it did, especially if you don't have a large screen, right?

<img width="1504" alt="Positron 2024-07-27 09 13 36" src="https://github.com/user-attachments/assets/f3caeb6d-3d46-455b-afc5-a1c0ae700b20">

**Reply by juliasilge** [reply]
Ah, I see, yep! This is somewhat related to #3746 in that it is about zooming or giving more screen real estate to some of our Positron-specific UI components. I will open an issue for a feature request. Thanks!

**Reply by juliasilge** [reply, marked as answer]
I started https://github.com/posit-dev/positron/issues/4179 to collect ideas on this, if you would like to share your perspective.
